mscore/events.cpp

```diff
@@ -698,7 +698,7 @@ void ScoreView::escapeCmd()
 
 static const char* stateName(ViewState s)
       {
-      const char* p;
+      std::string p;
```

Comment:
Ever since this change, the qDebug output in `ScoreView::changeState()` has been garbage. How about we change this to 
```
const char* p = "";
```
...